### DISCUSSION TITLE
How to use a specific version of R with quarto if multiple versions are installed.

### DISCUSSION THREAD

**Discussion by Fred-Wu** [quarto-dev/quarto-cli#3438] [Q&A] [answered]
I have two versions of R installed. One is V4.1.3, and the other one is V4.2.1 on my Windows machine. 

However, `quarto check` only points to V4.1.3 even after I delete the version number from the registry. 

How can I ask `quarto` to use V4.2.1 instead?

**Comment by mcanouil** [comment, marked as answer]
If you did install those R version using the default options, then the path on Windows are different.
I am assuming Quarto try to find R and uses the 4.1 and prior path.

Perhaps, there is an environment variable to be set (as for Python and other tools used by Quarto).
(Can't check the code base now from my smartphone.)


I would recommend to use something like "rig" (R installation manager): <https://github.com/r-lib/rig>.

**Reply by Fred-Wu** [reply]
Thanks for the advice. I have edited the Windows environment variable manually. Now it points to V4.2 correctly.
